bi_sql_editor/models/bi_sql_view.py

```diff
@@ -208,13 +208,20 @@ def button_set_draft(self):
                 # Drop ORM
                 sql_view._drop_model_and_fields()
 
-            sql_view.tree_view_id.unlink()
-            sql_view.graph_view_id.unlink()
-            sql_view.pivot_view_id.unlink()
-            sql_view.search_view_id.unlink()
-            sql_view.action_id.unlink()
-            sql_view.menu_id.unlink()
-            sql_view.rule_id.unlink()
+            if sql_view.tree_view_id:
```

Comment:
why did you added a test ? if rule_id is False, sql_view.r_le_id.unlink() make nothing.